scarb/tests/local_registry.rs

```diff
@@ -186,7 +186,7 @@ fn publish() {
             {
                 "v": "1.0.0",
                 "deps": [],
-                "cksum": "sha256:3771f51595f7df697a822823989714b3c9a83764a67fa517ac6d8df3cfc642bf",
+                "cksum": "sha256:8c612d3a5ec513259f786b692c91aa0f37d9d30e24428ae4d72dc7d39d00b45c",
```

Comment:
hmm why did checksums change?

Reply:
I think it's because of the `[dev-dependencies]` field now added to each `ProjectBuilder`

Reply:
I guess
https://github.com/software-mansion/scarb/blob/5cec6ef3de89d537880103c80856ae46fdec8540/scarb/src/core/publishing/manifest_normalization.rs#L41

Reply:
Can we avoid adding `[dev-dependencies]` in `ProjectBuilder` if not needed, then?